client/src/components/AutoHeightVirtualList.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { List } from "react-virtualized";
+
+export class AutoHeightVirtualList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.max_height = props.max_height || 400;
```

Comment:
Ah, missed this. Would be better as 
```
AutoHeightVirtualList.defaultProps = {
  max_height: 400,
}
```
and checking the current props `max_height` value in the `componentDidUpdate`.

As is, this component won't respond to changes in the `max_height` prop (not currently done, but could imagine it being done in a future use case).